fulp_modules/features/events/cateors/cateor.dm

```diff
@@ -50,10 +52,27 @@
 			playsound(src.loc, 'sound/effects/footstep/meowstep1.ogg', 25)
 
 /obj/effect/meteor/cateor/attack_hand(mob/living/thing_that_touched_the_cateor, list/modifiers)
-	to_chat(thing_that_touched_the_cateor, span_hypnophrase("How cuwious... CUWIOUS LIKE A-"))
+	to_chat(thing_that_touched_the_cateor, span_hypnophrase("How cuwious... CUWIOUS LIKE A CA—"))
 	Bump(thing_that_touched_the_cateor)
 
+/// Called 1.25 decisecond after a cateor calls 'Bump()' on a target.
+/// Global because cateors will be deleted by the time this is called, so it can't be a proc belonging
+/// to cateors themselves.
+/proc/cateor_hit_effects(mob/living/target)
+	target.extinguish() //Just in case it lit them on fire...
+	new /obj/effect/temp_visual/cateor_hit(get_turf(target))
+
 /obj/effect/meteor/cateor/Bump(mob/living/M)
```

Comment:
SINGLE LETTER VAR DETECTED IN FULP CODE, DEATH SQUAD GO

Reply:
***AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA*** (fixed)